.binder/postBuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# binder post build script
+set -ex
+
+(cd docs && make html)
+
+# uninstall docs requirements for a lighter docker image
+pip uninstall -y -r docs/requirements.txt
```

Comment:
what if there are requirements in this file that are necessary for the examples?

Reply:
Then I guess we will have to create a docs_only_requirements.txt 
For now I can put a comment in the requirements.txt saying "these are installed in binder, please don't put examples dependencies here"
I don't want to erase the env because the doc build installs the substra dependencies

Reply:
the `docs_only_requirements.txt` does not seem very maintainable, it would be hard for future devs to maintain this properly

good point about the env, if we were only in release mode then we could reinstall them from PyPi, but for dev purposes I guess we want to keep the installation from the main github branch

maybe we can do:
- either `pip install` the substra deps from the PyPi
- or `pip install git+https://github.com/Substra/substra`

But we need to check if this is really what takes the most time, might not be worth it

leads to speed up the launch time: https://discourse.jupyter.org/t/how-to-reduce-mybinder-org-repository-startup-time/4956

Reply:
I put stuff in [the summary](https://github.com/Substra/substra-documentation/pull/195#issuecomment-1266656023), it does matter to speed this up because that means that any user who opens an example for the first time waits 5-10min, and gets an error message 'this env is unusually long to launch, wait or kill?'

As I said, I am not fully convinced this is where we get the most speed up.

To better analyse the docker image, you can launch an env, do "show logs" and get the URL of the docker image

Reply:
Running binder for the first time could be part of the release process, so users don't run into it

Reply:
After a while the env has to be created again, it does not work to say that we build it once ourselves.